### PR TITLE
VAGOV-6144 Resolve git pre-commit issue

### PR DESCRIPTION
**Pull request by swirtSJW**
Fixes #623.

### CODE REVIEW

**Comment by swirtSJW** [comment]
Proof of working for a good branch name
image


Proof of catching a bad branch name:
image

File: hooks/git/pre-commit [hunk 1]
```diff
@@ -1,14 +1,14 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (ElijahLynn) [inline hooks/git/pre-commit]:
```suggestion
#!/usr/bin/env bash
```

Comment (swirtSJW) [reply hooks/git/pre-commit]: Tested this locally.  Works for me so adding it.
Bonus points for including the educational materials  :)

**Comment by ElijahLynn** [comment]
We just re-kicked off this build, it was stuck.

**Pull request merged**